website/docs/cluster-management/cluster-management-intro.mdx

```diff
@@ -12,7 +12,7 @@ In line with the mantra “cattle, not pets,” Weave GitOps Enterprise (WGE) si
 WGE fully supports a range of options, including:
 - [Crossplane integration](https://www.weave.works/blog/gitops-goes-beyond-kubernetes-with-weave-gitops-upbound-s-universal-crossplane)
 - Terraform integration, with a [Terraform Controller](https://docs.gitops.weave.works/docs/terraform/overview/) that follows the patterns established by Flux
-- Cluster API 
+- [Cluster API](https://cluster-api.sigs.k8s.io/)
```

Comment:
should these changes go in `/version-0.30.0` as well?

Reply:
Would be a good idea. I'll add.